clients/python/README.md

```diff
@@ -56,11 +56,40 @@ Owner and controller of cells in the game. As a programmer of an AI, you are a
             and a dumb AI will take over the player's cells until the player
             becomes active once again.
 
+### Cell
```

Comment:
I was asking myself the same question, and I figured that the players will skip the documentation as we ourselves do in competitions) especially since we want to make it an intense 8 hours, so might as well put the important concepts with the objects themselves... Not sure I like my approach.

Since we'll only work on a python client on this project, how about we put all the docs in here and create an issue for our documentation milestone to potentially split it up? Would you be okay with that?